Extensions/Today/TodayViewController.swift

```diff
@@ -119,7 +119,7 @@ class TodayViewController: UIViewController, NCWidgetProviding {
 
         let widgetView: UIView!
         self.extensionContext?.widgetLargestAvailableDisplayMode = .compact
-        let effectView = UIVisualEffectView(effect: UIVibrancyEffect.widgetPrimary())
+        let effectView = UIVisualEffectView(effect: UIVibrancyEffect.widgetEffect(forVibrancyStyle: .label))
```

Comment:
`widgetEffect` is only available on ios13, could you add an `#available` check and fallback to the old behaviour if on older OS?